store/src/lora_valid_poc.rs

```diff
@@ -86,9 +92,13 @@ impl TryFrom<LoraValidBeaconReport> for LoraValidBeaconReportV1 {
     type Error = Error;
     fn try_from(v: LoraValidBeaconReport) -> Result<Self> {
         let report: LoraBeaconReportReqV1 = v.report.into();
+        let location = match v.location {
```

Comment:
would `v.location.unwrap_or_else(|| "").to_string()` work?

Reply:
It does not, doesnt like the empty string:
mismatched types
expected `u64`, found `&str`